src/modules/sensors/vehicle_imu/VehicleIMU.cpp

```diff
@@ -546,8 +546,14 @@ bool VehicleIMU::Publish()
 
 	const Vector3f accumulated_coning_corrections = _gyro_integrator.accumulated_coning_corrections();
 
-	if (_accel_integrator.reset(delta_velocity, imu.delta_velocity_dt)
-	    && _gyro_integrator.reset(delta_angle, imu.delta_angle_dt)) {
+	uint32_t delta_vel_dt;
```

Comment:
Just to be safe I would either limit the configurable integration interval or bump the rest to uint32_t.